packages/sdk-react/src/ui/CounterpartCell/CounterpartCell.tsx

```diff
@@ -1,13 +1,8 @@
 import React, { useCallback } from 'react';
 
+import { components } from '@/api';
 import { useCounterpartById } from '@/core/queries';
-import {
-  PayableResponseSchema,
-  CounterpartResponse,
-  CounterpartType,
-  CounterpartOrganizationRootResponse,
-  CounterpartIndividualRootResponse,
-} from '@monite/sdk-api';
+import { PayableResponseSchema, CounterpartType } from '@monite/sdk-api';
```

Comment:
replace it with `components['schemas']['PayableResponseSchema']` pls

Reply:
also, replace the usage of `CounterpartType` with 'organization' below in current file